rest-model/src/main/java/org/jboss/pnc/rest/restmodel/TargetRepositoryRest.java

```diff
@@ -78,6 +78,15 @@
     @NotNull(groups = {WhenUpdating.class, WhenCreatingNew.class})
     private Set<Integer> artifactIds = new HashSet<>();
 
+    public TargetRepositoryRest(Integer id, Boolean temporaryRepo,
```

Comment:
I think, it is redundant as we have a Builder for this class.

Reply:
Fair point, will fix that!